src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -103,6 +103,43 @@ public ChainedHeader GetLastPowPosChainedBlock(IStakeChain stakeChain, ChainedHe
             return startChainedHeader;
         }
 
+        public Target GetNextTargetRequired(ChainedHeader chainedHeader, ChainedHeader lastPowPosBlock, ChainedHeader prevLastPowPosBlock, BigInteger targetLimit)
```

Comment:
1) As mentioned, i'd not pass CHs
2) first argument chainedHeader is not even used

Reply:
This method can actually just take one CH and becone specific to PoS